### PR TITLE
[REL] project_description to V11.

### PR DESCRIPTION
**Pull request by ValentinoDefelice**
[ADD] Models folder.
[FIX] Rename fields.
[FIX] Update README.
[FIX] Minor fix.

### CODE REVIEW

File: project_description/models/account_analytic_account.py [hunk 1]
```diff
@@ -10,6 +10,5 @@ class account_analytic_account(models.Model):
```
Comment (nicomacr) [inline project_description/models/account_analytic_account.py:10]: [Here](https://github.com/adhoc-dev/project/blob/7a30b818804954783bc2495d82582e3a5f0773b5/project_description/models/account_analytic_account.py#L8) the name of the model should be using the Camecase convencion

File: project_description/views/project_project_views.xml [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- INHERITED FORM -->
+    <record id="view_project_tags_project_form" model="ir.ui.view">
+        <field name="name">project_tags.project.form</field>
```
Comment (nicomacr) [inline project_description/views/project_project_views.xml]: The `name` should be `project.project.form.inherit`

File: project_description/views/project_project_views.xml [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <!-- INHERITED FORM -->
+    <record id="view_project_tags_project_form" model="ir.ui.view">
```
Comment (nicomacr) [inline project_description/views/project_project_views.xml]: The `id` should be `edit_project`

**Review by nicomacr — CHANGES_REQUESTED** [review]
Hi @ValentinoDefelice  thanks for the PR:
I put some comments below.

File: project_description/README.rst [hunk 4]
```diff
@@ -1,43 +1,42 @@
...
 ===================
 Project Description
 ===================
+
 Adds account_analytic_account description field on project form view.
 
 Installation
 ============
 
-To install this module, you need to:
```
Comment (zaoral) [inline project_description/README.rst]: This sections are empty, will be good if they have content as it is in the README guideline template.
Would be good if could look like the next one:

image

File: project_description/__init__.py [hunk 5]
```diff
@@ -3,4 +3,4 @@
 # directory
 ##############################################################################
 
-from . import project
+from . import models
```
Comment (zaoral) [inline project_description/__init__.py:6]: this file is marked as executable file and it should not.
Please remove the execute permissions with "chmod -x file_name"

Please this also apply to the next files

```
models/__init__.py
i18n/project_description.pot
views project_project_views.xml
```

File: project_description/models/account_analytic_account.py [hunk 6]
```diff
@@ -10,6 +10,5 @@ class account_analytic_account(models.Model):
 
     # para simplificar volvemos a crear el mismo campo que ya existia antes
     description = fields.Text(
-        'Description',
         help="Account analytic account description"
```
Comment (zaoral) [inline project_description/models/account_analytic_account.py]: >          help="Account analytic account description"

add `,` at the end of the line

>         help="Account analytic account description",

I think we could change the help attribute to "Description of the project"

File: project_description/README.rst [hunk 7]
```diff
@@ -1,43 +1,42 @@
...
+
+.. image:: https://img.shields.io/badge/license-AGPL--3-blue.png
+   :target: https://www.gnu.org/licenses/agpl
    :alt: License: AGPL-3
 
 ===================
 Project Description
 ===================
+
 Adds account_analytic_account description field on project form view.
```
Comment (zaoral) [inline project_description/README.rst]: This seems to be a technical premise.
Please add a more functional description, could be something like

"Be able to set a Description to Projects in order to extend information about the project meaning"

**Review by zaoral — CHANGES_REQUESTED** [review]
Hi @ValentinoDefelice 

Here are the results of the technical review you ask. Leave you some comments here.

About the functional review LGTM+1

**Comment by zaoral** [comment]
Last but not less, your PR is out-of-date with the base branch, please do a rebase to the 11.0 branch of your branch after apply the new suggested changes.

Best Regards

**Review by nicomacr — DISMISSED** [review]

**Review by zaoral — DISMISSED** [review]

**Review by nicomacr — APPROVED** [review]

**Pull request merged**